index.html

```diff
@@ -3916,8 +3865,8 @@ <h4>equivalentId</h4>
                 <dl>
                   <dt><dfn>equivalentId</dfn></dt>
                   <dd>
-The value of <code>equivalentId</code> MUST be a <a
-data-cite="INFRA#lists">list</a> where each item in the list is a <a
+The value of <code>equivalentId</code> MUST be an <a
+data-cite="INFRA#ordered-set">ordered set</a> where each item in the list is a <a
```

Comment:
This change looks good. I did a scan through the entire spec and we really don't need `list` anywhere... but that's not to say that the core data model shouldn't support it.